Control/lib/control-core/RequestHandler.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * @license
+ * Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+ * See http://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+ * All rights not expressly granted are reserved.
+ *
+ * This software is distributed under the terms of the GNU General Public
+ * License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+*/
+
+const {Log} = require('@aliceo2/web-ui');
+const log = new Log(`${process.env.npm_config_log_label ?? 'cog'}/controlrequests`);
+
+/**
+ * Handles AliECS create env requests
```

Comment:
I would suggest we keep the naming convention of Control.
I can see that 2 lines above the log will use `controlrequest` as label. The class itself is called `RequestHandler` while in the api file it is called `AliECSRequestHandler`

I would suggest we rename it `EnvironmentController` or `ControlEnvironmentControlleer` as it deals with requests for controlling an environment